src/components/game.jsx

```diff
@@ -249,16 +253,24 @@ class Game extends Component {
           if (this.isLastLevel()) {
             store.dispatch(gameOver('Игра окончена'));
             store.dispatch(testNewGame());
+            setTimeout(() => {
+              this.updateCurrentSession();
+              this.setState({
+                levelCompleted: true,
+              });
+            }, 2000);
```

Comment:
set timeout is better to move to separate const if they is not trivial (0 for just out of the loop, 400 is not immidiate, 800 can wait, other timeout value is better to move to separate constants )